### DISCUSSION TITLE
Airtable Update - PATCH or PUT

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#7275] [Help] [answered]
Hello!

Does anyone know if the **update** query for the airtable integration uses the PATCH or PUT API call in the background. I'm assuming PATCH is used as otherwise the PUT request will perform a destructive update and clear all unspecified cell values.

Thanks

**Comment by melohagan** [comment, marked as answer]
Hello @adamjdavies 

I believe it's a PATCH: https://help.landbot.io/article/s9r6j4iby4-how-to-create-update-delete-and-retrieve-records-in-airtable-post-patch-delete-get#2_update_a_record_patch

**Reply by ghost** [reply]
Perfect. Thank you!
